src/Codeception/Lib/Generator/Group.php

```diff
@@ -47,12 +47,12 @@ public function __construct($settings, $name)
     {
         $this->settings = $settings;
         $this->name = $name;
-        $this->namespace = $this->getNamespaceString($this->settings['namespace'] . '\\Group\\' . $name);
+        $this->namespace = $this->getNamespaceString($this->supportNamespace() . '\\Group\\' . $name);
     }
 
     public function produce()
     {
-        $ns = $this->getNamespaceString($this->settings['namespace'] . '\\' . $this->name);
+        $ns = $this->getNamespaceString($this->supportNamespace() . '\\' . $this->name);
```

Comment:
@DavertMik It could be a fault of my previous rebase, but `$ns` is unused now.
Do you remember what was the original intention?